docs/administrative-requirements.md

```diff
@@ -23,66 +23,61 @@ Preparing to connect your system to Enterprise Event Bus may require additional
+Teams that have an active ATO should review it to understand the implications of integrating with other systems.
 
 **For non-LHDI clients**:
 
-* If a prospective client system does not have a VASI ID, it will need to submit a LEAF intake request to get one. 
+ATO is evaluated once eMASS system registration has been completed and approved. Teams that do not have an active ATO should begin the process as early as possible in the Enterprise Event Bus integration process.
 
 **For new LHDI clients**:
 
-* For new teams on Lighthouse Delivery Infrastructure (LHDI), please consult your LHDI Enablement Liaison to start the LEAF intake request and LHDI-specific onboarding. 
+For new teams within the Lighthouse Delivery Infrastructure, please consult your LHDI Enablement Liaison to begin the ATO process.
 
-### eMASS
+## Consumers of BIP-sourced Events
 
-Enterprise Mission Assurance Support Service (eMASS) is VA’s Governance, Risk and Compliance (GRC) tool. All systems will be assessed in eMASS by the Risk Review team for an authorization recommendation to be submitted to the Authorizing Official (AO) for final ATO consideration.
+### OBI Access Form
 
-All VA systems are given an entry with a unique number in the eMASS system. The Enterprise Event Bus eMASS ID is 2350. 
+Event Bus Consumers consuming a BIP-sourced event that is available on the Event Bus must follow the OBI data access form process (aka the Corp DB Access form).
 
-Getting an eMASS ID is a prerequisite for System Categorization. 
+If the same consumer wants to consume additional BIP-sourced events at a later time, another OBI data access request form must be submitted to cover the use of that additional data.
 
-**For non-LHDI clients**:
+[Instructions for filling out the form (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Partner%20Teams/BIP%20Meetings%20and%20Materials/SOP_Corporate_DB_Access_Approval%20081123%201.pdf) and [an example (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Partner%20Teams/BIP%20Meetings%20and%20Materials/CorpDB%20Application%20Access%20Enterprise%20Event%20Bus%20full%20production.pdf) are available in the Enterprise Event Bus GitHub repository. 
 
-* If a prospective client system does not have an eMASS ID, it will need to submit a LEAF intake (formerly GRC intake) request: [Unified System Registry Intake (must be on VA network to view)](https://leaf.va.gov/NATIONAL/103/cybersecurity_request_portal/) to get one.
+Approval typically takes place in two stages: first for use in non-production environments, then for use in production.
```

Comment:
Should we mention that getting the second stage approval may not require submitting another request?

Reply:
Also, note that if the consumer will be using an event that is new to Event Bus, the Event Bus team will have to first fill out a Corp DB form to get approval for the event to be published to the bus, then the consumer team can request approval to consume the event from the Event Bus.

Reply:
No, there wouldn't be. I was suggesting that we make a note about that so that consumers are aware that there will be a pre-requisite step that Event Bus needs to take before they can make their own Corp DB request if the event they will be consuming is new to the Bus.

Reply:
I took a first pass at making these changes. Let me know what you think.

Reply:
Looks good - thanks!